packit.yaml

```diff
@@ -21,6 +21,7 @@ jobs:
     targets:
       - fedora-36
```

Comment:
Let's please drop fedora-36. We have traditionally only supported two parallel releases, there's too much overhead otherwise.

Reply:
Ok, I checked and Fedora 36 is supported till F 35. So do we stop releasing and testing or only testing?

Reply:
We stop both. (Logically, stop releasing first and then testing, but we can stop both at the same time in packit)